code/sql.c

```diff
@@ -7,13 +7,7 @@ CSQLInterface::CSQLInterface(void)
 {
 	connstate = STATE_INVALID;
 	result_set = nullptr;
-	Settings = Config("../config.json");
-	if (!Settings.isLoaded())
```

Comment:
@rezalas Was the `IsLoaded` check removed by mistake? Seems to undo a bit of #167.

Reply:
I realized after looking at the constructor for Config there was no way for this to ever return false if the object is instantiated.